packages/cashier-v2/src/index.scss

```diff
@@ -0,0 +1,12 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@import '@deriv/quill-design/dist/quill-design.css';
```

Comment:
Were you able to get quill-design and tailwind working here? Last time I tried, quill-tokens was overriding the styles from tailwind but overall I could use quill-tokens and some tailwind styles like `flex`

Reply:
Here's the PR for my implementation in case if your side didn't work:
https://github.com/binary-com/deriv-app/pull/12720

Reply:
Facing same issue:

Uploading Screen Recording 2024-01-17 at 12.32.12 PM.mov…